frontend/src/bundles/studio/components/timeline/rows/scenes-row.tsx

```diff
@@ -1,14 +1,30 @@
 import { Badge, Image } from '~/bundles/common/components/components.js';
 import { useAppSelector, useMemo } from '~/bundles/common/hooks/hooks.js';
+import { IconName, IconSize } from '~/bundles/common/icons/icons.js';
+import { Control } from '~/bundles/studio/components/components.js';
 import { RowNames } from '~/bundles/studio/enums/enums.js';
-import { setItemsSpan } from '~/bundles/studio/helpers/set-items-span.js';
+import {
+    getElementEnd,
+    setItemsSpan,
+} from '~/bundles/studio/helpers/helpers.js';
+import { useTimelineContext } from '~/bundles/studio/hooks/hooks.js';
 import styles from '~/framework/theme/styles/css-modules/timeline.module.css';
 
 import { Item, Row } from '../components.js';
 
 const ScenesRow: React.FC = () => {
     const scenes = useAppSelector(({ studio }) => studio.scenes);
     const scenesWithSpan = useMemo(() => setItemsSpan(scenes), [scenes]);
+    const { pixelsToValue } = useTimelineContext();
+
+    const scenesEnd = scenesWithSpan.at(-1)?.span.end ?? 0;
+    const buttonWidthInPixels = 60;
+
+    const buttonEnd = getElementEnd(
+        scenesEnd,
+        buttonWidthInPixels,
+        pixelsToValue,
+    );
```

Comment:
```suggestion
    const buttonEnd = getElementEnd(
        scenesEnd,
        pixelsToValue(buttonWidthInPixels)
    );
```
I think it is better not to pass pixelsToValue fn into getElementEnd, what do you think?